.github/workflows/dependency-review.yml

```diff
@@ -10,5 +10,39 @@ jobs:
     steps:
       - name: 'Checkout Repository'
         uses: actions/checkout@v4
-      - name: Dependency Review
+
+      - name: 'Dependency Review'
         uses: actions/dependency-review-action@main
+        id: review
+
+      - name: 'Check comment-content output'
+        if: always()
+        env:
+          COMMENT: ${{ steps.review.outputs.comment-content }}
+        run: |
+          test -n "$COMMENT"
+          echo "$COMMENT"
+
```

Comment:
☝️ it feels odd that the `comment-content` is HTML and the rest is JSON. Is this output needed. If so, should we present it in a more convenient form for use as an output?

Reply:
I'm not sure what you're suggesting. This output was added in https://github.com/actions/dependency-review-action/pull/698 and the use case was to post it in more places that just the PR comment. 

This change here is just for consistency with the rest of outputs. But if suggestion in https://github.com/actions/dependency-review-action/pull/707/files#r1524299054 is to remove this, this will be also gone.

Reply:
The HTML output is used for different purposes and we should be kept until we refactor how the summaries are generated, let's leave this here!